_docs/student/student_errors.md

```diff
@@ -0,0 +1,75 @@
+---
+category: Student
+title: Debugging Submitty Output
+---
+
+
+### Common user errors
+
+Your code is not tested locally first, Submitty should be used as the
+final submission server, not as a debugger. Before submitting anything
+make sure that your able to run your code locally without error.
+In addition the instructor may deciede to use hidden tests which still
+reveal the grade. Often these tests corner cases and exactly what those
+could be vary by your program but common cases to consider are positive, 
+negative, zero, and floating point values and how your code performs give
+unexpected but valid input.
```

Comment:
I am not sure if we really want to be so specific about what the corner cases are, as some programs may be unrelated to number operations. I am afraid that these hints may confuse someone working with some completely different problems (like assignments on string operations).